data/incipias/incipias first contact.txt

```diff
@@ -46,13 +46,15 @@ mission "Incipias: Warning of the Guardian"
 			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
 			label care
 			action
+				log `Entered a region of space northeast of the Hai. Was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit that region.`
```

Comment:
```suggestion
				log `Entered a region of space northeast of the Hai and was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit the region.`
```
Logs are a bit less "Point. Thing. Done." and more "The point, with the thing, and done," if that makes sense.
Also "that" makes more sense to be "the" since you're already referencing the region, not pointing at it as "that".